### PR TITLE
Chunter: archive channel

### LINKED ISSUE #1394: Chunter: Archive channel

### PR DESCRIPTION
**Pull request by budaeva**
Signed-off-by: budaeva <irina.budaeva@xored.com>

closes #1394

### CODE REVIEW

File: plugins/chunter-resources/src/index.ts [hunk 1]
```diff
@@ -80,6 +81,58 @@ export async function UnpinMessage (message: ChunterMessage): Promise<void> {
...
+
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        client.update(channel, { archived: true })
+        if (afterArchive)
+          afterArchive()
+
+        const loc = getCurrentLocation()
+
+        if (loc.path.includes(channel._id) ) {
+          const path = loc.path.slice(0, 2)
```
Comment (BykhovDenis) [inline plugins/chunter-resources/src/index.ts]: Why you change path so strange? You can just set `loc.path.length = 2`

Comment (budaeva) [reply plugins/chunter-resources/src/index.ts]: Fixed.
Added issue https://github.com/hcengineering/anticrm/issues/1431

File: plugins/chunter-resources/src/index.ts [hunk 2]
```diff
@@ -95,20 +95,12 @@ export async function ArchiveChannel (channel: Channel, afterArchive?: ()=>void)
...
-        if (loc.path.includes(channel._id) ) {
-          const path = loc.path.slice(0, 2)
-          path.push('0')
-        
-          const newloc = {
-            ...loc,
-            path
-          }
-          navigate(newloc)
+        if (loc.path.includes(channel._id)) {
```
Comment (BykhovDenis) [inline plugins/chunter-resources/src/index.ts]: You can check `loc.path[2] === channel._id` instead. It's faster

Comment (budaeva) [reply plugins/chunter-resources/src/index.ts]: Done

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**